tools/topology/topology2/include/pipelines/cavs/mixout-gain-eqiir-eqfir-drc-alh-dai-copier-playback.conf

```diff
@@ -0,0 +1,141 @@
+                                        num_input_audio_formats 1
+                                        num_output_audio_formats 1
+
+                                        # 32-bit 48KHz 2ch
+                                        Object.Base.input_audio_format [
+                                                {
+                                                        in_bit_depth		32
+                                                        in_valid_bit_depth	32
+                                                }
+                                        ]
+                                        Object.Base.output_audio_format [
+                                                {
+                                                        out_bit_depth		32
+                                                        out_valid_bit_depth	32
+                                                }
+                                        ]
+
+                                        Object.Control.bytes."1" {
+                                                IncludeByKey.EFX_SPK_FIR_PARAMS {
+                                                        "passthrough" "include/components/eqfir/passthrough.conf"
+                                                }
+                                        }
+                                }
+
+                                drc."1" {
+                                        num_input_audio_formats 1
+                                        num_output_audio_formats 1
+
+                                        # 32-bit 48KHz 2ch
+                                        Object.Base.input_audio_format [
+                                                {
+                                                        in_bit_depth		32
+                                                        in_valid_bit_depth	32
+                                                }
+                                        ]
+                                        Object.Base.output_audio_format [
+                                                {
+                                                        out_bit_depth		32
+                                                        out_valid_bit_depth	32
+                                                }
+                                        ]
+
+                                        Object.Control.bytes."1" {
+                                                IncludeByKey.EFX_SPK_DRC_PARAMS {
+                                                        "passthrough" "include/components/drc/passthrough.conf"
+                                                        "speaker_default" "include/components/drc/speaker_default.conf"
+                                                }
+                                        }
+                                }
+                        }
+
+			Object.Base {
+				!route [
+					{
+						source  mixout.$index.1
+						sink	gain.$index.1
```

Comment:
if you are just extending the routes in the base class, you don't need to include the existing routes @singalsu. So this can be removed

Reply:
If I remove it, the gain.21.1 isn't connected. The pipeline class mixout-gain-alh-dai-copier-playback doesn't contain routes. Should I add the missing route there and update the topologies those use it?

Reply:
I see it there: https://github.com/thesofproject/sof/blob/7718e435822745263c33b35def564438feb77157/tools/topology/topology2/include/pipelines/cavs/mixout-gain-alh-dai-copier-playback.conf#L95

Reply:
Oops, sorry you are right, how could I not see it (facepalm). But then what is wrong with my SubTreeCopy. Why isn't the route appearing from the base class?

Reply:
I also just checked that lack of route isn't just a tplgtool.py or tplgtool2.py issue, the playback in real device is not working if I remove the route add.

Reply:
This just pushed version works without duplicated route. @ranj063 found why it didn't work, the routes need to be defined as arrays in pipeline classes for subtreecopy usage.

Reply:
Thanks for fixing this!